vuepress/docs/next/docs/getting-started/entando-cli.md

```diff
@@ -0,0 +1,131 @@
+  - check-env            => Checks the environment for required dependencies and settings
+  - diag                 => Runs diagnostics and aggregates the related info in a .tgz file
+  - ent                  => Helper for managing the local ent installations
+  - ecr                  => Helper for managing the ECR
+  - help                 => Help information
+  - host                 => Helper for managing the system that hosts the quickstart VM
+  - jhipster           (not there)
+  - kubectl              => Helper for using kubectl in ent managed scenarios
+  - node-cmd-wrapper    => wrapper of node commands
+  - pod                  => Displays information related to a set of pods
+  - prj                 => Helper for managing Entando bundle projects
+  - profile              => Helper for managing an Entando App
+  - quickstart           => Helper for installing Entando instances locally
+  - run-tests            => Runs the internal tests
+> Further info about entando:
+  - ~/.entando/ent/v7.0.0/cli/v7.0.0/README.md
+  - https://www.entando.com/
+  - https://developer.entando.com/
+
+> ⚠ RECOMMENDED FIRST STEP ⚠ :
+  - Check the dependencies (ent check-env --help)
+```
+** New for version 7.1 **
+```
+~~~~~~~~~~~~~~~~~~~
+ ent bundle CLI
+~~~~~~~~~~~~~~~~~~~
+VERSION
+  @entando/entando-bundle-cli/0.0.1-ENG-3939-PR-101 darwin-x64 node-v14.19.0
+
+USAGE
+  $ ent bundle [COMMAND]
+
+TOPICS
+  api  Manage API claims
+  mfe  Micro Frontend operations
+  ms   Microservice operations
+  svc  Manage auxiliary services
+
+COMMANDS
+  build        Build bundle components
+  generate-cr  Generate the Entando Custom Resource (CR) for a bundle project
+  help         Display help for ent bundle.
+  info         Show status information for the bundle project
+  init         Perform the scaffolding of a bundle project
+  list         List the available components in the bundle
+  pack         Generate the bundle Docker images
+  publish      Publish bundle Docker images
+  run          Run bundle components
+
+```
+### Check the Environment
+
+The `check-env` command verifies the presence of required dependencies and the appropriate versions for your Entando instance, automatically installing dependencies with developer guidance. 
+```
+ent check-env develop
+```
+
+### Update the CLI
+To update ent to the latest version. Run `ent check-env develop` after all updates to check dependecies.
+
+``` sh
+bash <(curl -L "https://get.entando.org/cli") --update
+```
+>Alternatively, to perform a clean install, delete the `~/.entando` directory with `rm -rf ~/.entando`. Then reinstall the CLI with the instructions above. This will also remove the private copies of JHipster, Entando Blueprint, etc.
+
+### Customize Quickstart
+The `ent quickstart` command supports parameters to customize your quickstart environment. These options allow you to modify specific properties of your VM, installation versions and databases.
```

Comment:
strictly speaking a parameter is not an option. "It includes options to modify..."